Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/buttons/ShowMoreButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useCallback, useState} from 'react';
+import {LinkButton} from "./Button";
+import {MdExpandLess, MdExpandMore} from "react-icons/md";
+
+import styles from './ShowMoreButton.module.scss'
+interface IShowMoreButtonProps {
+    onClick: () => void;
+}
+
+const ShowMoreButton = ({
+    onClick,
+} : IShowMoreButtonProps) => {
+    const [ showMore, setShowMore] = useState(false);
+    
+    const buttonText = showMore
+        ? "Show less"
+        : "Show more";
+    
+    const buttonIcon = showMore
+        ? <MdExpandLess/>
+        : <MdExpandMore/>;
+    
+    const onShowMoreClick = useCallback(
```

Comment:
`on...` is the naming for **props**
`handle...` is the naming for handling.
This is handling, so ... `handleShowMoreClick`